neurots/morphmath/sample.py

```diff
@@ -145,3 +145,13 @@ def ph(phs, random_generator=np.random):
     """Samples randomly a persistence diagram from the input distribution."""
     index = random_generator.choice(len(phs))
     return phs[index]
+
+
+def sample_spherical_unit_vectors(rng):
```

Comment:
Isn't it the same as `morphmath.utils.get_random.point`? I mean, it's not the same method but the objective is the same, right?

Reply:
yes, it's probably the same, but mine is cleaner/faster ;P